cmd/gen-go-sample/main_test.go

```diff
@@ -289,6 +289,35 @@ func TestWriteFile(t *testing.T) {
 
 }
 
+func TestEnum(t *testing.T) {
```

Comment:
I suggest changing `"EType"` and `"AType"` in `initTestGenerator()` to be more descriptive. I assume "E" is for enum. What is "A" for?

Reply:
I think `AType` is just "a type". It is a very ordinary message.